plugin/helpers_bots.go

```diff
@@ -9,6 +9,25 @@ import (
 	"github.com/pkg/errors"
 )
 
+type ensureBotOptions struct {
+	ProfileImagePath string
+	IconImagePath    string
+}
+
+type Option func(*ensureBotOptions)
```

Comment:
This should also be renamed to
```suggestion
type ensureBotOption func(*ensureBotOptions)
```